File: spec/javascript/test_enum_0.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('Enum0', 'src/enum_0.bin', function(r, Enum0) {
    assert.equal(r.pet1, Enum0.Animal.CAT);
    assert.equal(r.pet2, Enum0.Animal.CHICKEN);
});

File: spec/python/test_enum_0.py
import unittest

from enum_0 import Enum0

class TestEnum0(unittest.TestCase):
    def test_enum_0(self):
        r = Enum0.from_file("src/enum_0.bin")

        self.assertEquals(r.pet_1, Enum0.Animal.cat)
        self.assertEquals(r.pet_2, Enum0.Animal.chicken)
